docs/README

```diff
@@ -0,0 +1,6 @@
+Yeah, there's ruby stuff in here. You can ignore every bit of it unless you'd
```

Comment:
ruby???

Reply:
please see the rest of the file :) jekyll is ruby. if you want to use jekyll to review the changes you've made, you can. or you don't have to.